tests/Stokes-IB/test0/main.cpp

```diff
@@ -269,8 +269,10 @@ class StokesIBSolver : public IBAMR::StaggeredStokesSolver
 
         // Initialize Stokes op and IB FAC pc
         {
-            Pointer<SAMRAIVectorReal<NDIM, double> > u_p = PETScSAMRAIVectorReal::getSAMRAIVector(x);
-            Pointer<SAMRAIVectorReal<NDIM, double> > f_g = PETScSAMRAIVectorReal::getSAMRAIVector(b);
+            Pointer<SAMRAIVectorReal<NDIM, double> > u_p;
```

Comment:
@amneetb Did you really mean to make these changes (u_p, f_g, etc are left uninitialized)?

Reply:
@boyceg : I changed it because of your API change to PETScSAMRAIVectorReal ;-). It is initialized at the next line

PETScSAMRAIVectorReal::getSAMRAIVector(x, &u_p);